### PR TITLE
Update app.src

### PR DESCRIPTION
**Pull request by vincentux**
Update 0.05.2

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/lufi_ynh%20PR44%20(ericgaspar)/)

### CODE REVIEW

**Comment by alexAubin** [comment]
Sounds like a legit change to me ?

File: manifest.json [hunk 1]
```diff
@@ -6,15 +6,15 @@
 		"en": "Self hosting files and sharing anonymous application",
 		"fr": "Application d'hébergement et de partage de fichiers anonyme"
 	},
-	"version": "0.03.5~ynh2",
+	"version": "0.05.2~ynh3",
```
Comment (ericgaspar) [inline manifest.json:9]:
```suggestion
	"version": "0.05.2~ynh1",
```

**Review by ericgaspar — CHANGES_REQUESTED** [review]
As we increment the version number...

**Comment by ericgaspar** [comment]
Outdated with #47

**Pull request closed without merging**